### PR TITLE
added pdf-icon component 

### PR DESCRIPTION
**Pull request by kkoryaka**
This one related to issue https://github.com/cagov/design-system/issues/113 and https://github.com/cagov/cannabis.ca.gov/issues/74

### CODE REVIEW

File: components/pdf-icon/src/index.scss [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  outline: 1px solid;
+  outline-offset: -2px;
+  padding: 0;
+  position: relative;
+  left: 1px;
+  top: -3px;
+  line-height: .9rem; 
+  text-decoration: none;
+  svg path {
+      fill: var(--primary-color, #064e66);
```
Comment (aaronhans) [inline components/pdf-icon/src/index.scss:13]: nice!

File: components/pdf-icon/src/index.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+var pdf = '<span class="pdf-link-icon" aria-hidden="true"><svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"width="25.1px" height="13.6px" viewBox="0 0 25.1 13.6" style="enable-background:new 0 0 25.1 13.6;" xml:space="preserve"><path d="M11.7,9.9h1.5c1.7,0,3.1-1.4,3.1-3.1s-1.4-3.1-3.1-3.1h-1.5c-0.3,0-0.6,0.3-0.6,0.6v4.9c0,0.2,0.1,0.3,0.2,0.4C11.4,9.9,11.6,9.9,11.7,9.9L11.7,9.9z M12.3,5h0.9c1,0,1.8,0.8,1.8,1.8s-0.8,1.8-1.8,1.8h-0.9V5z"/><path d="M17.8,9.9c0.2,0,0.3-0.1,0.4-0.2c0.1-0.1,0.2-0.3,0.2-0.4V7.5h1.2c0.3,0,0.6-0.3,0.6-0.6c0-0.3-0.3-0.6-0.6-0.6h-1.2V5h2.1c0.3,0,0.6-0.3,0.6-0.6c0-0.3-0.3-0.6-0.6-0.6h-2.8c-0.3,0-0.6,0.3-0.6,0.6v4.9c0,0.2,0.1,0.3,0.2,0.4C17.5,9.9,17.7,9.9,17.8,9.9L17.8,9.9z"/><path d="M6.2,9.9c0.2,0,0.3-0.1,0.4-0.2c0.1-0.1,0.2-0.3,0.2-0.4V8.1H8c1.2,0,2.1-1,2.1-2.1c0-1.2-1-2.1-2.1-2.1H6.2c-0.3,0-0.6,0.3-0.6,0.6v4.9c0,0.2,0.1,0.3,0.2,0.4C5.9,9.9,6,9.9,6.2,9.9L6.2,9.9z M9,6c0,0.3-0.1,0.5-0.2,0.7C8.5,6.8,8.3,6.9,8,6.9H6.8V5H8c0.2,0,0.5,0.1,0.7,0.2C8.9,5.5,9,5.7,9,6L9,6z"/><path d="M5,9.3c0,0.8-1.2,0.8-1.2,0C3.8,8.5,5,8.5,5,9.3z"/></svg></span><span class="sr-only"> (this is a pdf file)</span>';
```
Comment (aaronhans) [inline components/pdf-icon/src/index.js:2]: we need to encapsulate these variables 

you can wrap it in an IIFE https://developer.mozilla.org/en-US/docs/Glossary/IIFE then we'll avoid creating global vars if somebody uses this by itself

**Review by aaronhans — CHANGES_REQUESTED** [review]
Excellent Kontantin!

I added one recommendation about wrapping javascript

File: components/pdf-icon/src/index.js (hunk 2 above)
Comment (kkoryaka) [reply components/pdf-icon/src/index.js:4]: just encapsulated it into the function

**Review by aaronhans — APPROVED** [review]
cool

**Pull request merged**